package com.revature.exceptions;

public class InternalError extends Exception {
	public InternalError() {
		super("There was an internal error");
	}

}
